CauldronMods/Controller/Heroes/Echelon/Cards/SurpriseAttackCardController.cs

```diff
@@ -130,5 +133,14 @@ private IEnumerator ChangeDamageTypeFromEffectResponse(AddStatusEffectAction se)
                 }
             }
         }
+
+        public override CustomDecisionText GetCustomDecisionText(IDecision decision)
+        {
+
+            return new CustomDecisionText($"Do you want to change all damage dealt from the status effect made by {PowerSourceTitleForCustomText} to Psychic?", $"Should they change all damage dealt from the status effect made by {PowerSourceTitleForCustomText} to Psychic?", $"Vote for if they should change all damage dealt from the status effect made by {PowerSourceTitleForCustomText} to Psychic?", "change damage to Psychic");
```

Comment:
I'm not sure how to trigger this dialog to test it. I can see it is called on line 35, but can't parse out how to make that happen.

Reply:
That bit is called when the power sets up a continuing status effect that can cause damage - something like Road Warrior Mainstay or Omnitron-U's innate powers.